packages/components/help-text/stories/help-text.stories.js

```diff
@@ -0,0 +1,34 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.help-text';
+
+storiesOf('ts-help-text', module)
+	.add(
+		'Title & messages',
+		() => html`
+			<ts-help-text
+				messages='["Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias, at atque deserunt, dignissimos dolor doloribus eaque eius eos eum excepturi fuga, hic incidunt itaque molestias necessitatibus pariatur rem repellat soluta", "message example 1", "message example 2"]'
+				title="Title"
+				size="small"
+			></ts-help-text>
+		`
+	)
+	.add(
+		'Single message',
+		() => html`
+			<ts-help-text
+				messages='["Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias, at atque deserunt, dignissimos dolor doloribus eaque eius eos eum excepturi fuga, hic incidunt itaque molestias necessitatibus pariatur rem repellat soluta"]'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 239 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ALineLength)